google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -759,7 +760,16 @@ def query_job query, params: nil, priority: "INTERACTIVE", cache: true,
         #   dialect. When set to true, the values of `large_results` and
         #   `flatten` are ignored; the query will be run as if `large_results`
         #   is true and `flatten` is false. Optional. The default value is
-        #   false.
+        #   true.
+        # @param [Boolean] legacy_sql Specifies whether to use BigQuery's
+        #   [legacy
+        #   SQL](https://cloud.google.com/bigquery/docs/reference/legacy-sql)
+        #   dialect for this query. If set to false, the query will use
+        #   BigQuery's [standard
+        #   SQL](https://cloud.google.com/bigquery/docs/reference/standard-sql/)
+        #   When set to false, the values of `large_results` and `flatten` are
```

Comment:
That text is taken from the [API documentation](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.useLegacySql). Every time we get fancy with stuff like that we live to regret it. I'd prefer to leave the warnings to the documentation.